packages/scripts/src/commands/app-data/convert/processors/flowParser/parsers/generator.parser.ts

```diff
@@ -23,13 +20,23 @@ export class GeneratorParser extends DefaultParser<FlowTypes.GeneratorFlow> {
       return;
     }
     try {
-      this.flow._generated = this.generateFlows(flow, dataListRows);
-      // this.handleOutputs(generated);
+      const generated = this.generateFlows(flow, dataListRows);
```

Comment:
Am I right in thinking that the author could have specified a name for the output flow that is identical to the name of another output flow (or pre-existing flow) and that it wouldn't have been picked up until this point? In that case, the most recently processed flow with that name would overwrite the previous one without throwing any errors, so it could be worth handling here

Reply:
Created https://github.com/IDEMSInternational/open-app-builder/issues/2451 for future follow-up

Reply:
Great, thanks for making the issue. I hadn't considered the case of intentionally overwriting a flow with a generated one, so good to have that possibility captured